source/Util/VTK_Util.hpp

```diff
@@ -0,0 +1,63 @@
+#pragma once
+
+#ifdef USE_VTK
+
+#include <vtkFloatArray.h>
+#include <vtkSmartPointer.h>
+#include <Eigen/Dense>
+
+/**
+ * @brief Converts a vector of doubles to a vtkFloatArray
+ *
+ * @param name The desired name of the vtkFloatArray
+ * @param data The std::vector of data
+ * @return vtkSmartPointer<vtkFloatArray>
+ */
+vtkSmartPointer<vtkFloatArray>
+vector_to_vtk_array(const char* name, const std::vector<double>& data);
+
+/**
+ * @brief Returns the number of rows in an Eigen::Matrix that has 1 column
+ *
+ * @tparam T
+ * @tparam N
+ * @param mat
+ * @return constexpr unsigned int
+ */
+template<typename T, unsigned int N>
```

Comment:
The [compilation failed again](https://github.com/FloatingArrayDesign/MoorDyn/actions/runs/5003644216/jobs/8982274314). Apparently on Linux you need the specialization

```
template<typename T, int N>
constexpr unsigned int
get_vec_size(const Eigen::Matrix<T, N, 1>& mat)
{
	return N;
}
```

I can even remove the `unsigned int` one. Probably is best to keep both

Reply:
Oh, very true! Eigen has the enigmatic `Eigen::VectorXd`...